controllers/grid/CustomLocaleGridHandler.inc.php

```diff
@@ -71,8 +72,11 @@ function updateLocale($args, $request) {
 			// save changes
 			$changes = $args['changes'];
 
+			/* UZH CHANGE OJS-72 2019/02/26/mb */
+			/* $customFilesDir = Config::getVar('files', 'public_files_dir') .
+				"/presses/$contextId/" . CUSTOM_LOCALE_DIR . "/$locale"; */
 			$customFilesDir = Config::getVar('files', 'public_files_dir') .
-				"/presses/$contextId/" . CUSTOM_LOCALE_DIR . "/$locale";
+                                "/journals/$contextId/" . CUSTOM_LOCALE_DIR . "/$locale";
```

Comment:
This still refers to `journals`, rather than the app-agnostic directory name.

Reply:
(Also, watch out for space-based indents -- it looks like some have been added here instead of tabs.)

Reply:
vi. Sigh ... . Will look out.